README.md

```diff
@@ -1,10 +1,10 @@
 # decotra
 
-Save a intermediate product and upload to S3.
+This tool saves intermediate objects such as models files in machine learning project s and upload them to S3.
 
-`Decotra`'s main feature is concreted with decorator, so it is easy to code and to remove.
+The main features of `Decotra` is provided with Python decorator, and therefore we can easy to add them to the codes and remove them.
 
-In my use case, tracking many results of functions on server and upload to S3 (I use minio), analyzing the calculation process ad-hoc.
```

Comment:
Without conjunction, two sentences are not connected in English. The following change merged two sentences into one.